lib/compat/wordpress-6.2/script-loader.php

```diff
@@ -80,7 +80,10 @@ function gutenberg_resolve_assets_override() {
 		'wp-edit-blocks',
 	);
 
-	if ( current_theme_supports( 'wp-block-styles' ) ) {
+	if (
+		current_theme_supports( 'wp-block-styles' ) &&
+		( ! is_array( $editor_styles ) || count( $editor_styles ) === 0 )
+	) {
```

Comment:
This needs to be updated in Core in [wp-includes/script-loader.php](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/script-loader.php#L1627).